projects/aca-content/src/lib/components/sidenav/components/expand-menu.component.html

```diff
@@ -10,36 +10,30 @@
       mat-button
       class="action-button full-width"
     >
-      <adf-icon *ngIf="item.icon" [value]="item.icon"></adf-icon>
       <span class="action-button__label">{{ item.title | translate }}</span>
     </button>
   </div>
 </ng-container>
 
 <ng-container *ngIf="item.children && item.children.length">
   <mat-expansion-panel
-    [expanded]="acaExpansionPanel.hasActiveLinks()"
+    [expanded]="(isRouteAbout() || isRouteProfile() || isRouteSearch()) ? !acaExpansionPanel.hasActiveLinks() : acaExpansionPanel.hasActiveLinks()"
```

Comment:
I was saying this hundred of times already, please create 1 function in the component that evaluates the state, don't move the code to the HTML as you are not then testing it properly. 

`[expanded]="isItemExpanded(item)`

also, you make every panel expanded for all the children, what are you trying to achieve here?